doc/proxy-metrics.md

```diff
@@ -131,5 +133,61 @@ request_total{
 }
 ```
 
+# Transport-Level Metrics
+
+The following metrics are collected at the level of the underlying transport
+layer.
+
+### `tcp_accept_open_total`
+
+A counter of the total number of transport connections which have been accepted
+by the proxy.
+
+### `tcp_accept_close_total`
+
+A counter of the total number of transport connections accepted by the proxy
+which have been closed.
+
+### `tcp_connect_open_total`
+
+A counter of the total number of transport connections which have been opened
+by the proxy.
+
+### `tcp_connect_close_total`
+
+A counter of the total number of transport connections opened by the proxy
+which have been closed.
+
+### `tcp_connections_open`
+
+A gauge of the number of transport connections currently open.
+
+### `sent_bytes`
+
+A counter of the total number of sent bytes.
+
+### `received_bytes`
+
+A counter of the total number of recieved bytes.
+
+### `tcp_connection_duration_ms`
+
+A histogram of the duration of the lifetime of a connection, in milliseconds.
+
+## Labels
+
+Each of these metrics has the following labels:
+
+* `direction`: `inbound` if the connection was established either from outside the
+                pod to the proxy, or from the proxy to the application,
+               `outbound` if the connection was established either from the
+                application to the proxy, or from the proxy to outside the pod.
+* `protocol`: `http` if the connection corresponds to an HTTP/1 or HTTP/2 request,
```

Comment:
We decided in the implementation PR to not include this label.

Reply:
Yeah, I removed it from `simulate-proxy` in #822 and will remove it from the docs as well.